### ISSUE TITLE
Area of effect damage from Permanent Immolation sources needs rework

### ISSUE THREAD

**Issue by nvs** [nvs/gem#55]
Or at least I'm pretty sure they don't stack. By buff auras I mean the AoE damage auras attached to Star Ruby, Uranium, etc. I'm almost positive they do not stack with a second copy of the same structure. It could be possible they do not stack with other buff auras from different structures either. Need to test that.

**Comment by nvs** [comment]
One possible solution is to have these structures share their kill buffs together. So even if the damage cannot stack, their damage could cover a larger area. Just an idea. Still, having them just simply stack would be the ideal solution.

**Comment by nvs** [comment]
Also, they don't improve with damage, as mentioned in #241.

**Comment by nvs** [comment]
Further testing shows that all of these are based upon 'Permanent Immolation', and, as a result, they all break in odd ways. For reference they use the following values:

- Blood Star, 0.20 seconds, 10 damage
- Rose Quartz Crystal, 1.00 seconds, 50 damage
- Uranium 238, 0.50 seconds, 100 damage

Note that no other sources (e.g. Viper Slate or Star Ruby) were tested. However, what seems to be happening is that these damage sources are fighting to apply their damage. Basically, some seem to have precedence over the others. For example, Rose Quartz Crystal is applied instead of Uranium 238 in all cases. However, when in the range of the Blood Star, the Rose Quartz Crystal Damage does not apply. Instead, only 10 damage per second is done. Somehow the damage duration of the Rose Quartz Crystal is adopted by the Blood Star, and it ends up dealing less damage.

In summary, it's broken and the entire concept of area of effect damage applied in this fashion needs to be scrapped.

**Comment by nvs** [comment]
Need to test various damage types to verify what bypasses armor reduction.

**Comment by nvs** [comment]
This may have to wait for a more thorough approach to balancing, as is being considered in #131.
